### ISSUE TITLE
running a task once per deployment (rather than node restart)

### ISSUE THREAD

**Issue by benfoxall** [theodi/open-data-certificate#726]
The `odc:deploy` runs after each node [is restarted](https://github.com/theodi/odi-chef/blob/master/roles/certificate.rb#L11-L13).

This means the task can run multiple times per deploy (once from each web node).

Is it possible to have another task that runs only once per deploy?  So we could split the task into `odc:after_restart` and `odc:after_deploy`.

Related to #725
